Source/ColoredMoodBar/HarmonyPatch.cs

```diff
@@ -153,36 +154,22 @@ public class MoodPatch {
         }
 
         private static Texture2D getMoodTexture(ref Pawn colonist) {
-            float statValue = colonist.GetStatValue(StatDefOf.MentalBreakThreshold, true);
             float currentMoodLevel = colonist.needs.mood.CurLevel;
-            Texture2D moodTexture = null;
-
-            // Extreme break threshold
-            if (currentMoodLevel <= statValue) {
-                moodTexture = Main.extremeBreakTex;
-            }
-            // Major break threshold
-            else if (currentMoodLevel <= statValue + 0.15f) {
-                moodTexture = Main.majorBreakTex;
-            }
-            // Minor break threshold
-            else if (currentMoodLevel <= statValue + 0.3f) {
-                moodTexture = Main.minorBreakTex;
-            }
-            // Neutral
-            else if (currentMoodLevel <= 0.65f) {
-                moodTexture = Main.neutralTex;
+            MentalBreaker pawnState = colonist.mindState.mentalBreaker;
+
+            if (pawnState.BreakExtremeIsImminent) {
+                return Main.extremeBreakTex;
+            } else if (pawnState.BreakMajorIsImminent) {
+                return Main.majorBreakTex;
+            } else if (pawnState.BreakMinorIsImminent) {
+                return Main.minorBreakTex;
+            } else if (pawnState.CurMood < .65f) {
+                return Main.neutralTex;
+            } else if (pawnState.CurMood < .9f) {
+                return Main.contentTex;
```

Comment:
I can't remember why I had both a neutral and content color tbh. Was there any significance to the difference? I just see the different mental break thresholds and the >90% threshold for a chance of inspirations on https://www.rimworldwiki.com/wiki/Mood.

Reply:
Yeah that’s the only useful reason. Splitting them is nice IMO so you can tell which pawns can pick up inspos and don’t need any mood boosts vs which ones are just neutral and are candidates for mood influencers